components/sup/src/manager/mod.rs

```diff
@@ -613,6 +629,19 @@ impl Suitability for SuitabilityLookup {
     }
 }
 
+fn obtain_process_lock(fs_cfg: &FsCfg) -> Result<()> {
+    match OpenOptions::new().write(true).create_new(true).open(&fs_cfg.proc_lock_file) {
+        Ok(_) => Ok(()),
+        Err(_) => Err(sup_error!(Error::NoProcessLock)),
```

Comment:
Yep, makes sense to me--it makes the file potentially actionable and has a way to determine if a stale file is legit or not.